presto-hive/pom.xml

```diff
@@ -57,6 +57,11 @@
             <artifactId>hudi-hadoop-mr</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
```

Comment:
Hbase is a pretty big dependency and this could cause issues.

Reply:
Hudi uses HBase in two ways:
1. The metadata files are written as HFile.
2. To maintain index of Hudi filegroup id to external bootstrap file.
In this PR, we just need `Bytes` class from hbase-common for initializing `HoodieTableFileSystemView` and I have excluded deps that we don't need. We have created a thinner [hudi-presto-bundle](https://github.com/apache/hudi/pull/3623) which I plan to update in presto-hive  pom in a subsequent PR.

Reply:
Currently Presto does not pull in other bigger open source jars (hadoop, hive) and the only way to do is to fork the code reduce it to only bare minimal code and then take a dependency on it. 

https://github.com/prestodb/presto-hadoop-apache2
https://github.com/prestodb/presto-hive-apache

Hbase will cause dependency hell for Presto as some of the dependencies are conflicting. https://mvnrepository.com/artifact/org.apache.hbase/hbase-common/2.4.8

This will be a blocker to merge this PR.

Reply:
Even linked [PR](https://github.com/apache/hudi/pull/3623/files), shades everything which will avoid dependency issues but it will cause the jar size to explode. 

What is the size of the shaded jar ?

Reply:
It's about [13 MB](https://mvnrepository.com/artifact/org.apache.hudi/hudi-presto-bundle/0.10.0). 
Unfotunately, HFile is only packaged as part of HBase. Would it be more preferable to use hudi-presto-bundle instead of adding hbase directly?

Reply:
@arunthirupathi Can you please review this again? Check the latest commit where i'm using the stripped down hudi-presto-bundle.

Reply:
For presto-hive can you please do a dependency tree diff of before and after and paste what changed ?

mvn dependency:tree -pl presto-hive (run this on master and the change branch)

Reply:
Below is the diff. The full dependency tree is in [this gist](https://gist.github.com/codope/cb0d174d371e8be698d0342f5cf3393f).
```
sagars@Sagars-MacBook-Pro presto % git diff --no-index presto_hive_master_dep.txt presto_hive_hudi_dep.txt
diff --git a/presto_hive_master_dep.txt b/presto_hive_hudi_dep.txt
index 3f6e69f26a..21733b24f4 100644
--- a/presto_hive_master_dep.txt
+++ b/presto_hive_hudi_dep.txt
@@ -20,8 +20,7 @@
 [INFO] +- com.facebook.presto:presto-expressions:jar:0.268-SNAPSHOT:compile
 [INFO] +- com.facebook.presto:presto-cache:jar:0.268-SNAPSHOT:compile
 [INFO] |  \- org.alluxio:alluxio-shaded-client:jar:2.7.0:compile
-[INFO] +- org.apache.hudi:hudi-common:jar:0.9.0:compile
-[INFO] +- org.apache.hudi:hudi-hadoop-mr:jar:0.9.0:compile
+[INFO] +- org.apache.hudi:hudi-presto-bundle:jar:0.10.0:compile
 [INFO] +- com.facebook.presto:presto-memory-context:jar:0.268-SNAPSHOT:compile
 [INFO] +- com.facebook.presto:presto-hive-common:jar:0.268-SNAPSHOT:compile
 [INFO] +- com.facebook.presto:presto-hive-metastore:jar:0.268-SNAPSHOT:compile
@@ -249,6 +248,6 @@
 [INFO] ------------------------------------------------------------------------
 [INFO] BUILD SUCCESS
 [INFO] ------------------------------------------------------------------------
-[INFO] Total time:  2.809 s
-[INFO] Finished at: 2021-12-23T12:51:07+05:30
+[INFO] Total time:  2.857 s
+[INFO] Finished at: 2021-12-23T12:52:44+05:30
 [INFO] ------------------------------------------------------------------------
```